### PR TITLE
Update CRC Cards and Progress Report

### PR DESCRIPTION
**Pull request by shawnpl7**

### CODE REVIEW

File: phase1/CRC_Cards.md [hunk 1]
```diff
@@ -1 +1,167 @@
...
+## Entities
+
+Class Name: User  
+Parent Class: N/A  
+Responsibilities: stores the user id, username, password, HashMap mapping cuisine to number of likes user put for that cuisine, a String bio, a list of followers, a list of Users the User is following, a list of Posts the User has Posted. Contains user actions such as addFollower.  
+Collaborators: UserProfilePresenter, UserManager
+
+Class Name: Recipe  
+Parent Class: N/A  
+Responsibilities: stores a recipe id, list of steps, a list of ingredients, and the title of the recipe. Basic functions like deleteStep.
```
Comment (yowang20) [inline phase1/CRC_Cards.md:12]: Recipe doesn't have deleteStep anymore at least not right now

File: phase1/CRC_Cards.md [hunk 2]
```diff
@@ -1 +1,167 @@
...
+Collaborators: User, Post, Ingredient, CountableIngredient, MeasurableIngredient  
+
+Class Name: Ingredient  
+Parent Class: N/A  
+Responsibilities: stores the name of the Ingredient  
+Collaborators: Recipe, MeasurableIngredient, CountableIngredient
+
+Class Name: MeasurableIngredient  
+Parent Class: Ingredient  
+Responsibilities: stores amount of ingredient, stores type of measurement, getter functions for the amount of ingredient and the type of measurement
```
Comment (yowang20) [inline phase1/CRC_Cards.md:22]: and name of ingredient

File: phase1/CRC_Cards.md [hunk 3]
```diff
@@ -1 +1,167 @@
...
+Collaborators: Recipe, MeasurableIngredient, CountableIngredient
+
+Class Name: MeasurableIngredient  
+Parent Class: Ingredient  
+Responsibilities: stores amount of ingredient, stores type of measurement, getter functions for the amount of ingredient and the type of measurement  
+Collaborators: Ingredient
+
+Class Name: CountableIngredient  
+Parent Class: Ingredient  
+Responsibilities: stores number of ingredient, gets the number of ingredient
```
Comment (yowang20) [inline phase1/CRC_Cards.md:27]: and name of ing

**Review by ericjest — APPROVED** [review]

File: phase1/CRC_Cards.md [hunk 4]
```diff
@@ -1 +1,167 @@
...
+Collaborators: User, DatabaseManager
+
+Class Name: FeedManager  
+Parent Class: N/A  
+Responsibilities: handle post recommendations/obtain posts from people user follows, set the filter on the Feed  
+Collaborators: Post, User, Feed
+
+Class Name: PostManager  
+Parent Class: N/A  
+Responsibilities: Create new Ingredients, create a new Post, create a new Recipe, comment/like a Post
```
Comment (yowang20) [inline phase1/CRC_Cards.md:72]: get post info

File: phase1/CRC_Cards.md [hunk 5]
```diff
@@ -1 +1,167 @@
...
+Collaborators: UserProfile, UserManager, RecipeAppController
+
+Class Name: FeedController  
+Parent Class:N/A  
+Responsibilities: gets Feed related inputs (setting the filter, liking a post, commenting on a post), stores a FeedManager which handles Feed related actions, convert Feed to a displayable output type, sends Feed related output  
+Collaborators: FeedManager, PostManager, Feed, User, RecipeAppController
+
+Class Name: PostController  
+Parent Class:N/A  
+Responsibilities: gets post creation related inputs (recipe steps, ingredients, category) and displays the appropriate outputs
```
Comment (yowang20) [inline phase1/CRC_Cards.md:133]: interact with post

**Review by yowang20 — CHANGES_REQUESTED** [review]

**Pull request merged**